README.md

```diff
@@ -661,6 +661,25 @@ npm run build -- --with-deno
 DENO_DEPLOY_TOKEN=... deployctl deploy --project=... --prod dist/serve.js --exclude node_modules
 ```
 
+### AWS Lambda (experimental)
+
+you have to add the [`hono`](https://hono.dev/getting-started/aws-lambda) package to your project:
```

Comment:
hono is already a dependency, so we don't need to install it, or no?

Reply:
you are right - I tested again and it works without - did remove the info.